excellent/tests.go

```diff
@@ -109,6 +107,37 @@ var _ utils.VariableResolver = XTestResult{}
 // Tests
 //------------------------------------------------------------------------------------------
 
+// IsStringEQ returns whether two strings are equal (case sensitive).
+//
+//  @(is_string_eq("foo", "foo")) -> true
+//  @(is_string_eq("foo", "foo")) -> true
```

Comment:
repeat?

Reply:
Sometimes nice to define what the TestResult will return in these examples as well. (I would guess it is the matched string)